config/purposes/scrna_core_cdna_prep.yml

```diff
@@ -144,3 +144,11 @@ LRC GEM-X 5p cDNA PCR CITE:
   :asset_type: plate
   :stock_plate: false
   :input_plate: false
+  :presenter_class: Presenters::PermissiveSubmissionPlatePresenter
+  :submission_options:
+    CITE-Seq Branch - Automated Submission:
+      template_name: 'Limber-Htp - scRNA Core cDNA PCR CITE'
+      allowed_extra_barcodes: false
+      request_options:
+        library_type: 'Chromium single cell GEM-X 5p v3 cell surface protein'
+        request_type_key: limber_scrna_core_cdna_pcr_cite
```

Comment:
Please specify the library type in the Sequencescape request type definition. Sequencescape:
`limber_scrna_core_cdna_pcr_cite` in config/default_records/request_types/014_limber_scrna_core_cdna_prep_request_types.yml

See config/default_records/request_types/015_limber_scrna_core_library_prep_request_types.yml as an example.